_data/simple-icons.json

```
@@ -9189,7 +9189,7 @@
         },
         {
             "title": "Pets at Home",
-            "hex": "4BA840",
+            "hex": "04A928",
```

```suggestion
            "hex": "00AA28",
```
Color picked from wordmark was different

---

Applied suggestion